README.md

```diff
@@ -40,10 +40,151 @@ Note: If you use the Drush command, it is advisable to Move (not copy) the insta
 
 ## Configuration
 
+### Djatoka Image Server
 Set the paths for 'Djatoka server base URL' and configure OpenSeadradon in Administration » Islandora » OpenSeadragon (admin/islandora/module).
 
 ![Configuration](https://camo.githubusercontent.com/c1bf991b5cc758a4420444564a91b286007e6f6e/687474703a2f2f692e696d6775722e636f6d2f4e6566597169432e706e67)
 
+If you have an *existing* install it's required to update Openseadragon to it's latest version. You can do this quickly 
+with the provided Drush command.
+
+```bash
+drush openseadragonplugin
+```
+
+### Loris Image Server
+
+The [Loris Image server](https://github.com/loris-imageserver/loris) currently only works when configured to directly 
+communicate with Fedora. As such it does not work with Islandora's authentication system, meaning it can only be used to 
+display public datastreams, or it can be configured to give full access to show anything in the system. This does not 
+give any user the ability to access any datastream however, just the JPEG generated for display in the browser, not the 
+original.
+
+Additionally there are issues with it's caching system in that it needs to be manually flushed please read the project 
+[documentation](https://github.com/loris-imageserver/loris/blob/development/doc/cache_maintenance.md) for more 
+information.
+
+Provided below is instructions for installing loris on Ubuntu 14.04, the commands / packages may change slightly 
+depending on your setup.
+
+#### Create loris user
```

Comment:
The documentation around resolver integration w/ Fedora is definitely light. A PR is most welcome.

As for caching...yeah; cron scripts are all we have. They're in the repo but to not get deployed automatically because they're pretty lightweight and everyone kept wanting to change them to suit their local environment. This should be covered in the docs already, but is probably worth calling out in any upstream documentation as well.

BTW. I'm not sure how much longer Loris will have a backend cache at all, TBH. Something in front (varnish, squid, whatev.) make a lot more sense to me these days. If you have thoughts/opinions about that I'd be happy to hear them.